apisix/plugins/limit-count/limit-count-redis-cluster.lua

```diff
@@ -112,5 +113,5 @@ function _M.incoming(self, key)
     return 0, remaining, ttl
 end
 
-
+_M.script = script
```

Comment:
If you want to export this, it's better to put it in init.lua, don't refer to each other between parallel files. But it doesn't seem appropriate to put it in init.lua, so let's keep it as it is

Reply:
I mean don't export the script, keep it as it is

Reply:
Ah sorry let me fix it.

Reply:
FYI, we can't test the script without exporting it, so I removed the tests as well.